projects/transfer-journey/src/lib/constants/dynamic-translations.ts

```diff
@@ -0,0 +1,5 @@
+import '@angular/localize/init';
+
+export const TRANSLATIONS = {
+  makeTransferTitle: $localize`:make transfer title|header of the page@@make-transfer-journey.header.text:Make a transfer`,
+} as const;
```

Comment:
Also, why do we need this?

Reply:
this is what I commented during the meeting and the reason for that is the handling of translations

Reply:
Sorry, that i was not explicit,  i mean type casting specifically. i have tried without `as const` and no warnings or errors appeared.

Reply:
yes, at the end I applied all of your suggestions. Including:
-lint
-use of unknown
-remove "as const"